src/main/java/com/google/devtools/build/lib/analysis/IncompatiblePlatformProvider.java

```diff
@@ -0,0 +1,35 @@
+// Copyright 2020 The Bazel Authors. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.devtools.build.lib.analysis;
+
+import com.google.devtools.build.lib.concurrent.ThreadSafety.Immutable;
+
+/**
+ * Provider instance for the {@link target_compatible_with} attribute.
+ *
+ * The presence of this provider is used to indicate that a target is incompatible with the current
+ * platform. Any target that provides this will automatically be excluded from {@link
+ * SkyframeAnalysisResult}'s list of configured targets.
```

Comment:
How do I validate that these links are working? I can't find the generated docs for this.

Reply:
You can click on links at cs.opensource.google/bazel (like [this](https://cs.opensource.google/bazel/bazel/+/master:src/main/java/com/google/devtools/build/lib/analysis/ConfiguredTargetFactory.java;l=85?q=file:ConfiguredTargetFactory.java&ss=bazel%2Fbazel)). But that only happens after you've submitted the code.  I'm okay with that.

Reply:
Okay. I'll fall back to "best effort" then. Thanks!